etna/search/views.py

```diff
@@ -90,17 +90,16 @@ def get_buckets_for_display(
         return bucket_list
 
     def get_context_data(self, **kwargs):
-        if self.bucket_list:
-            buckets = self.get_buckets_for_display(
-                self.api_result.bucket_counts, self.current_bucket_key
-            )
+        buckets = self.get_buckets_for_display(
+            self.api_result.bucket_counts, self.current_bucket_key
+        )
 
-            # Set this to True if any buckets have results
-            buckets_contain_results = False
-            for bucket in buckets:
-                if bucket.result_count:
-                    buckets_contain_results = True
-                    break
+        # Set this to True if any buckets have results
+        buckets_contain_results = False
+        for bucket in buckets:
+            if bucket.result_count:
+                buckets_contain_results = True
+                break
```

Comment:
Can we revert these `get_context_data()` changes if they aren't actually needed?

Reply:
The condition [if self.bucket_list](https://github.com/nationalarchives/ds-wagtail/blob/99f0561faa27ae37274a78737ecdac5d281a3114/etna/search/views.py#L96-L111): is dropped as 
- The [return ](get_context_data)assigns uninitialized  values in context when if condition is used.
- the `bucket_list` is assigned for search views [ex.](https://github.com/nationalarchives/ds-wagtail/blob/99f0561faa27ae37274a78737ecdac5d281a3114/etna/search/views.py#L586)

Reply:
Okay. But as a rule, we should be aiming to keep changes as minimal as possible. Not only does it reduce unnecessary risk and make change history much easier to track... It makes review less open to misinterpretation.